docs/source/release/v3.13.0/diffraction.rst

```diff
@@ -78,6 +78,8 @@ Single Crystal Diffraction
 
 - New algorithm :ref:`IntegratePeaksProfileFitting <algm-IntegratePeaksProfileFitting>` to integrate peaks using 3D profile fitting in reciprocal space.
 
+- New algorithm :ref:`RefineSatellitePeaks <algm-RefineSatellitePeaks>` to predict the location of fractional satellite peaks using a set of nuclear peaks and a set of seed satellite peaks.
```

Comment:
Why is this algorithm called `RefineSatellitePeaks` when it is doing prediction, We do commonly use the term `Predict` in Mantid Algorithms such as `PredictPeaks`?

Reply:
My thinking was this: I didn't go with `Predict` because this algorithm requires the user to supply some hand picked seed peaks, so in a way you are already enforcing what you expect, more so than just from crystallography theory. The other point is that this does more than simply predict where peaks are, it also integrates them to filter any peaks that obviously don't exist so what you get back should be actual, real peaks in your data.

However, I'm not that happy about this name either and can see the counter argument. I am on the fence about whether it is worth the change.